gateways/orbital/orbital.go

```diff
@@ -0,0 +1,122 @@
+package orbital
+
+import (
+	"bytes"
+	"encoding/xml"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/BoltApp/sleet"
+	"github.com/BoltApp/sleet/common"
+)
+
+type Credentials struct {
+	username   string
+	password   string
+	merchantID int
+}
+
+type OrbitalClient struct {
+	host        string
+	credentials Credentials
+	httpClient  *http.Client
+}
+
+func NewClient(env common.Environment, credentials Credentials) *OrbitalClient {
+	return &OrbitalClient{
+		host:        orbitalHost(env),
+		credentials: credentials,
+		httpClient:  common.DefaultHttpClient(),
+	}
+}
+
+func (client *OrbitalClient) Authorize(request *sleet.AuthorizationRequest) (*sleet.AuthorizationResponse, error) {
+
+	authRequest := buildAuthRequest(request)
```

Comment:
lets just send the credentials down into buildAuthRequest so we can do it there?